docs/airgap.md

```diff
@@ -1,14 +1,19 @@
-# Airgap support
+# Airgap support and deployment
 
 :::tip
-This page concerns only official XO Appliances, and not XO built from sources.
+This page is relevant only to official XO Appliances and does not cover XO built from sources.
```

Comment:
Unrelated changes.

This documentation has already been reviewed and corrected by @Danp2 and @Fohdeesha

Reply:
Yes, but my impression was a suboptimal phrasing originally. I made various tries with ChatGPT and this one was the most clear to me. I do think the word relevant is far more… relevant in this context, and the rest of the sentence makes it more clear too.

Reply:
It's not about splitting the changes, it's about fixing something that was already validated by two native English speakers.

I have asked @Danp2 to do a pass on this, he will fix as he see fit.